pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
```

Comment:
> this still does not seem right. we need to call it client secret to make it consistent with etcd configuration.

Let's call it "clientSecret".

>  it seems to me that the operator can generate key/cert for itself from the provided member/client secret anyway.

Generate cert needs CA cert and **key**. We don't pass in **key** currently.